### PR TITLE
MVE_New_Form_Proof_Of_Concept

### PR DESCRIPTION
**Pull request by Santino-Trade**

### CODE REVIEW

File: barriers/models/barriers.py [hunk 1]
```diff
@@ -364,6 +364,27 @@ def latest_programme_fund_progress_update(self):
             return self.programme_fund_progress_updates[0]
         return None
 
+    @property
+    def start_date(self):
+        if self.data.get("start_date") is not None:
```
Comment (chris-pettinga) [inline barriers/models/barriers.py:369]: I generally prefer positive conditionals but that may just be may be. Also good case for walrus operator.

```suggestion
        if start_date := self.data.get("start_date"):
```

File: barriers/models/barriers.py [hunk 2]
```diff
@@ -364,6 +364,27 @@ def latest_programme_fund_progress_update(self):
...
+            return dateutil.parser.parse(self.data.get("start_date"))
```
Comment (chris-pettinga) [inline barriers/models/barriers.py:370]: Then here use the variable assigned in the conditional.

```suggestion
            return dateutil.parser.parse(start_date)
```

**Review by ferozerub — APPROVED** [review]

File: templates/reports/barrier_status_wizard_step.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                            {% for help_text in form.status.help_text %}
+                                {% if help_text.0 == value %}
+                                    <span id="{{ name }}_hint" class="govuk-hint govuk-radios__hint">
+                                        {{ help_text.1 }}
+                                    </span>
+                                {% endif %}
+                            {% endfor %}
+                        </div>
+
+                        {% if forloop.counter == 2 %}
```
Comment (chris-pettinga) [inline templates/reports/barrier_status_wizard_step.html:53]: this feels a little messy and liable to break. Something to add to the refactoring ticket

File: templates/reports/done_wizard_step.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+{% block fields %}
+    <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                <h1 class="govuk-fieldset__heading">
+                    DONE
+                </h1>
+            </legend>
+            <p class="govuk-body">
+                Hopefully.
```
Comment (chris-pettinga) [inline templates/reports/done_wizard_step.html:14]: is this meant to be here?

**Review by chris-pettinga — APPROVED** [review]
Left some general feedback but approving for times sake

**Pull request merged**